### DISCUSSION TITLE
Shell rupture in latest version of OpenRadioss

### DISCUSSION THREAD

**Discussion by AlexRiot1** [OpenRadioss/OpenRadioss#1647] [Q&A]
Hi everyone !
I've just downloaded the latest compiled version of OpenRadioss, and launched an old model that was working with the previous version. However now, I'm encountering a lot of issues with shell element rupture that is not supposed to happen (and was not happening in the previous version). Does anyone encounter similar issues ? Thanks for your feedback !
Alex

**Comment by MarianBulla** [comment]
Hi Alex,
this is hard to judge about the root cause, without knowing the model.
Can you add a small example of your model, please ?

Thanks and best regards,
Marian

**Comment by AlexRiot1** [comment]
Hi Marian,

I've reverted to an older version, and my models work once more. However, I still don't understand how this is possible with a more recent version of OpenRadioss... I've also noticed there was a big difference of size between the current version we're using (119 MB), and the version currently available online (79 MB), are there chunks missing, or did the software go through a big step of optimization ?
I'd very much like to give you an idea of the model I ran on both the old (11/2022) and the new (06/2023). Unfortuntely, it's part of a project inside my company. Is this channel just between the 2 of us, or anyone can have access to it ?

Thanks in advance, and sorry for the delay.
Alex

**Reply by MarianBulla** [reply]
Hi Alex,
yes, the code was reorganized in a larger way. Also some components are not automatically included in OpenRadioss (as, for example: encryption).
The differnces, can also come through some bug-fixes or other changes in the code, which can be explained, one haveing a deeper look into the model.

This is an open chanell, so please, do NOT send any confidential info through this chanel.

Instead of this, you can contact my via e-mail.

Thanks for understanding and best regards,
Marian
